docroot/modules/custom/mass_metatag/tests/src/ExistingSite/CollectionsMetadataTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\Tests\mass_metatag\ExistingSite;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\paragraphs\Entity\Paragraph;
+use Drupal\taxonomy\Entity\Vocabulary;
+use Drupal\Tests\mass_metatag\Traits\TestContentTrait;
+
+/**
+ * Collections metadata tests.
+ */
+class CollectionsMetadataTest extends MetadataTestCase {
+
+  use TestContentTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getContent(): ContentEntityInterface {
+    $label_term = $this->createTerm(Vocabulary::load('label'), [
+      'name' => 'TestTermLabel',
+    ]);
+    $collection_term = $this->createTerm(Vocabulary::load('collections'), [
+      'name' => 'TestTermCollection',
+      'field_reusable_label' => [$label_term],
+      'field_url_name' => 'test-collection-label'
+    ]);
+
+    return $collection_term;
+  }
+
+  /**
+   * Asserts that a collections page produces the metadata we expect.
+   */
+  public function testHasExpectedMetadata() {
+    $entity = $this->getContent();
```

Comment:
```suggestion
    $entity = $this->createContent();
```

Reply:
Renaming this causes issues, we should go deeper and rename the base method and everywhere it is used. So reverting to what it was